components/SummaryStats/SummaryStats.tsx

```diff
@@ -35,40 +26,18 @@ const [useSummaryStats] = bind(
               value: formatter?.(statValue) ?? statValue,
             } as IRow
           }),
-        })),
+        ),
       ),
     ),
   ),
-  {} as SummaryStatsViewModel,
+  {} as IRow[],
```

Comment:
If you're casting an object as an array, something's gone wrong

Reply:
```suggestion
  [] as IRow[],
```